src/containers/Careers.js

```diff
@@ -15,25 +19,122 @@ const CareersContainer = styled.div`
   flex-direction: column;
 `;
 
+const SwingWrapper = styled.div`
+  position: relative;
+  height: 70%;
+  width: 80%;
+  margin: 0 2rem;
+  z-index: 0;
+  overflow: visible;
+`;
+
+let swipeCount = 1;
```

Comment:
I really feel like this should be in state. Kinda the point of using state in the first place.

Reply:
Done :+1: